request/builder.js

```diff
@@ -76,6 +76,66 @@ class CertificateRegistrationRequestBuilder {
   }
 }
 
+/**
+ * Used to build a FunctionRegistrationRequest.
+ */
+class FunctionRegistrationRequestBuilder {
+  /**
+   * @constructor
+   * @param {FunctionRegistrationRequest} request
+   * @param {SignatureSigner} signer
+   */
+  constructor(request, signer) {
+    this.request = request;
+    this.signer = signer;
+  }
+
+  /**
+   * Sets the ID of the certificate holder
+   * @param {string} id
+   * @return {FunctionRegistrationRequestBuilder}
+   */
+  withFunctionId(id) {
+    /** @const */
+    this.functionId = id;
+    return this;
+  }
+
+  /**
+   * Sets the certificate version
+   * @param {string} name
+   * @return {FunctionRegistrationRequestBuilder}
+   */
+  withFunctionBinaryName(name) {
+    /** @const */
+    this.functionBinaryName = name;
+    return this;
+  }
+
+  /**
+   * Sets the functionBytes of the certificate
+   * @param {string} functionBytes
+   * @return {FunctionRegistrationRequestBuilder}
+   */
+  withFunctionByteCode(functionBytes) {
+    /** @const */
+    this.functionByteCode = functionBytes;
+    return this;
+  }
+
+  /**
+   * Builds the FunctionRegistrationRequest
```

Comment:
```suggestion
   * Build the FunctionRegistrationRequest
```

Reply:
Wait, there is no `s`? I thought it's going to be `it Builds`?

Reply:
I think `Builds` is correct.

Reply:
Ok